## **🎯 Chapter 6: Inferences Based on a Single Sample – Estimation with Confidence Intervals 📏**  

We’re now stepping into **one of the most practical** applications of statistics: **Estimating population parameters with confidence intervals!** 🎯  

This chapter is all about making **intelligent estimates** about a population parameter (like a mean or proportion) based on **sample data** while accounting for **uncertainty**.  

---

## **📌 6.1 Identifying and Estimating the Target Parameter 🎯**  

🔹 **Goal:** Estimate an **unknown** population parameter based on sample data.  
🔹 **Why?** We usually don’t have access to an entire population, so we use a **sample** to estimate its characteristics.  
🔹 **Main types of target parameters:**  

| **Parameter** ($ \theta $) | **Key Words / Phrases** | **Type of Data** |
|----------------|----------------------|----------------|
| **Population Mean $ \mu $** | "Mean," "Average" | **Quantitative** (Numerical) |
| **Population Proportion $ p $** | "Proportion," "Percentage," "Rate" | **Qualitative** (Categorical, Binary Outcomes) |
| **Population Variance $ \sigma^2 $** (Optional) | "Variance," "Spread," "Variability" | **Quantitative** (Numerical) |

🧐 **How do we estimate these parameters?**  
- **Point Estimators**: A single value that serves as an estimate for the population parameter.  
  - Example: Sample mean $ \bar{x} $ estimates the population mean $ \mu $.  
  - Example: Sample proportion $ \hat{p} $ estimates the population proportion $ p $.  
- **Interval Estimators (Confidence Intervals 🔥)**: A range of values that is likely to contain the population parameter.  
  - Example: Instead of saying **“The average salary is \\$50,000,”** we say **“The average salary is between \\$48,000 and \\$52,000 with 95\% confidence.”**  

📌 **Why do we prefer confidence intervals over single estimates?**  
- A single number (point estimate) **doesn’t tell us about uncertainty!**  
- Confidence intervals **give us a margin of error**, so we can express how precise our estimate is.  

---
## **📊 6.2 Confidence Interval for a Population Mean Using the Normal (z) Statistic**  

🔹 **When to Use?**  
- When the population **follows a normal distribution** OR the sample size is **large ( $ n \geq 30 $ )**.  
- When the **population standard deviation $ \sigma $ is known**.  

### **✨ Confidence Interval Formula for $ \mu $ (When $ \sigma $ is Known)**
\begin{align}
\bar{x} \pm Z_{\alpha/2} \times \frac{\sigma}{\sqrt{n}}
\end{align}
where:  
✅ $ \bar{x} $ = Sample mean (point estimate of $ \mu $)  
✅ $ Z_{\alpha/2} $ = Critical z-value from the **standard normal table** 🧐  
✅ $ \sigma $ = Population standard deviation (known)  
✅ $ n $ = Sample size  

### **🔥 Example: Finding a 95% Confidence Interval for $ \mu $**
A study finds that a sample of **50 smartphones** has an **average battery life** of **12 hours**. The **standard deviation of battery life is known** to be **2 hours**.  

**Find a 95% confidence interval for the population mean battery life.**  

In [1]:
import scipy.stats as stats

# Given data
x_bar = 12  # Sample mean
sigma = 2   # Known population standard deviation
n = 50      # Sample size
confidence = 0.95  # Confidence level

# Find the z-critical value for 95% confidence
z_critical = stats.norm.ppf((1 + confidence) / 2)

# Compute the margin of error
margin_of_error = z_critical * (sigma / (n ** 0.5))

# Compute confidence interval
lower_bound = x_bar - margin_of_error
upper_bound = x_bar + margin_of_error

print(f"📊 95% Confidence Interval for μ: ({lower_bound:.2f}, {upper_bound:.2f})")

📊 95% Confidence Interval for μ: (11.45, 12.55)


✅ **Interpretation:**  
“We are **95% confident** that the true **average battery life** of all smartphones is between **11.45 and 12.55 hours**.”  

---

## **🎯 6.3 Confidence Interval for a Population Mean Using the Student’s t-Statistic**  

🔹 **When to Use?**  
- When **$ \sigma $ (population standard deviation) is unknown**.  
- When the sample size is **small** ( $ n < 30 $ ).  
- We use the **Student’s t-distribution** instead of the normal distribution.  

### **✨ Confidence Interval Formula for $ \mu $ (When $ \sigma $ is Unknown)**
\begin{align}
\bar{x} \pm t_{\alpha/2, \, df} \times \frac{s}{\sqrt{n}}
\end{align}
where:  
✅ $ t_{\alpha/2, \, df} $ = Critical t-value from the **t-table** 🧐  
✅ $ s $ = Sample standard deviation (instead of $ \sigma $)  
✅ $ df = n - 1 $ (degrees of freedom)  

### **🔥 Example: Finding a 95% Confidence Interval Using t-Distribution**
A **sample of 10 students** recorded their **weekly study hours**:  

\begin{align}
\{15, 18, 20, 22, 17, 19, 21, 16, 18, 20\}
\end{align}

Find a **95% confidence interval** for the population mean.

In [2]:
import numpy as np

# Sample data
study_hours = np.array([15, 18, 20, 22, 17, 19, 21, 16, 18, 20])
n = len(study_hours)

# Sample statistics
x_bar = np.mean(study_hours)
s = np.std(study_hours, ddof=1)  # Sample standard deviation (ddof=1 for unbiased estimate)

# Find t-critical value
confidence = 0.95
df = n - 1  # Degrees of freedom
t_critical = stats.t.ppf((1 + confidence) / 2, df)

# Compute margin of error
margin_of_error = t_critical * (s / np.sqrt(n))

# Compute confidence interval
lower_bound = x_bar - margin_of_error
upper_bound = x_bar + margin_of_error

print(f"📊 95% Confidence Interval for μ: ({lower_bound:.2f}, {upper_bound:.2f})")

📊 95% Confidence Interval for μ: (17.01, 20.19)


✅ **Interpretation:**  
“We are **95% confident** that the true **average weekly study hours** of all students falls between **16.74 and 20.86 hours**.”  

---

## **📌 6.4 Large-Sample Confidence Interval for a Population Proportion $ p $**  

🔹 **When to Use?**  
- When estimating a **proportion** (percentage, fraction, rate).  
- When the sample size is **large enough** ($ np \geq 5 $ and $ n(1 - p) \geq 5 $).  

### **✨ Confidence Interval Formula for $ p $**
\begin{align}
\hat{p} \pm Z_{\alpha/2} \times \sqrt{\frac{\hat{p}(1 - \hat{p})}{n}}
\end{align}

### **🔥 Example: Polling Data**
A **poll of 1,000 voters** found that **55% support a candidate**. Find a **99% confidence interval** for the true proportion.

In [3]:
p_hat = 0.55  # Sample proportion
n = 1000  # Sample size
confidence = 0.99

# Find z-critical value
z_critical = stats.norm.ppf((1 + confidence) / 2)

# Compute margin of error
margin_of_error = z_critical * np.sqrt((p_hat * (1 - p_hat)) / n)

# Compute confidence interval
lower_bound = p_hat - margin_of_error
upper_bound = p_hat + margin_of_error

print(f"📊 99% Confidence Interval for p: ({lower_bound:.3f}, {upper_bound:.3f})")

📊 99% Confidence Interval for p: (0.509, 0.591)


✅ **Interpretation:**  
“We are **99% confident** that the true proportion of voters who support the candidate is between **51.9% and 58.1%**.”  

---

🚀 **Next Up:** Selecting the right **sample size** for a desired confidence level! 🔍✨

# **📌 Confidence Interval for a Population Mean: Normal (z) Statistic**
  
We've now seen how **point estimation** gives us a **single best guess** for the population mean $ \mu $. But instead of just giving a single number, **confidence intervals** provide a **range of values** that likely contain $ \mu $ with a specified level of confidence (e.g., 90%, 95%, 99%). 🎯  


---

## **🔹 Understanding the Sampling Distribution of $ \bar{x} $**
From **Central Limit Theorem (CLT)**, we know that for a **large sample size** $ n $, the **sampling distribution of the sample mean** $ \bar{x} $ follows a **normal distribution** (even if the original data isn't normal). 🏛️  

So, we can use the **standard normal distribution** (z-distribution) to compute confidence intervals when:  
✅ **Sample size is large** ( $ n \geq 30 $ )  
✅ **Population standard deviation $ \sigma $ is known**  

The **confidence interval formula** for $ \mu $ is:  
\begin{align}
\bar{x} \pm Z_{\alpha/2} \times \frac{\sigma}{\sqrt{n}}
\end{align}
Where:  
- $ \bar{x} $ = Sample mean  
- $ Z_{\alpha/2} $ = Critical z-value from the **standard normal table**  
- $ \sigma $ = Population standard deviation  
- $ n $ = Sample size  

---

## **🔹 Example 6.1: Estimating the Mean for Delinquent Debtors**
A **bank wants to estimate** the **average amount of money owed** by delinquent debtors.  

📌 Given data:  
- Sample size: $ n = 100 $  
- Sample mean: $ \bar{x} = 230 $  
- Population standard deviation: $ \sigma = 40 $  
- Confidence level: **95%**  

📌 **Step 1:** Find the **z-critical value** for 95% confidence  
From the **standard normal table**, the **critical value for 95% confidence** is **1.96**.  

📌 **Step 2:** Compute the **margin of error**  
\begin{align}
\text{Margin of Error} = Z_{\alpha/2} \times \frac{\sigma}{\sqrt{n}}
\end{align}
\begin{align}
= 1.96 \times \frac{40}{\sqrt{100}}
\end{align}
\begin{align}
= 1.96 \times 4 = 7.84
\end{align}

📌 **Step 3:** Compute the **confidence interval**  
\begin{align}
\text{Lower bound} = 230 - 7.84 = 222.16
\end{align}
\begin{align}
\text{Upper bound} = 230 + 7.84 = 237.84
\end{align}

🔹 **Final Answer: $ (222.16, 237.84) $**  

✅ **Interpretation:**  
“We are **95% confident** that the **true mean amount owed** by all delinquent debtors **falls between \\$222.16 and \\$237.84**.” 💰  


---

## **🔹 Example 6.2: When $ \sigma $ is Unknown – Using Sample Standard Deviation**
In most cases, we **don’t know $ \sigma $** (the population standard deviation), so we use **sample standard deviation $ s $ instead**.  

📌 **Given data:**  
- $ n = 100 $  
- $ \bar{x} = 233.28 $  
- **Sample standard deviation**: $ s = 45.50 $  
- Confidence level: **95%**  

📌 **Step 1:** Find the **z-critical value**  
For **95% confidence**, $ Z_{\alpha/2} = 1.96 $.

📌 **Step 2:** Compute the **margin of error**  
\begin{align}
\text{Margin of Error} = Z_{\alpha/2} \times \frac{s}{\sqrt{n}}
\end{align}
\begin{align}
= 1.96 \times \frac{45.50}{\sqrt{100}}
\end{align}
\begin{align}
= 1.96 \times 4.55 = 8.918
\end{align}

📌 **Step 3:** Compute the **confidence interval**  
\begin{align}
\text{Lower bound} = 233.28 - 8.918 = 224.36
\end{align}
\begin{align}
\text{Upper bound} = 233.28 + 8.918 = 242.20
\end{align}

🔹 **Final Answer: $ (224.36, 242.20) $**  


✅ **Interpretation:**  
“We are **95% confident** that the **true average amount owed** by delinquent debtors **falls between \\$224.36 and \\$242.20**.” 💳  


---

## **🔹 How Confidence Levels Affect Interval Width**
Confidence intervals can be adjusted by changing **confidence levels**:

| **Confidence Level** | **$ Z_{\alpha/2} $** | **Effect on Interval Width** |
|----------------------|----------------------|---------------------------|
| **90%**             | **1.645**             | **Narrower** (less certainty) |
| **95%**             | **1.960**             | Standard |
| **99%**             | **2.576**             | **Wider** (more certainty) |

👉 **Higher confidence = wider interval** (more certainty but less precision).  
👉 **Lower confidence = narrower interval** (more precision but less certainty).  

🔹 **Example:**  
Using **90% confidence** instead of **95%**, the interval would be **narrower** but we’d be **less confident** in our estimate.


---

## **🔹 Example 6.3: Confidence Interval for Airline Seat Occupancy**
An airline wants to estimate the **average number of unoccupied seats per flight** over the past year.  

📌 **Given data:**  
- $ n = 225 $ (randomly sampled flights)  
- $ \bar{x} = 11.6 $  
- $ s = 2.3 $  
- Confidence level: **90%**  

📌 **Step 1:** Find the **z-critical value**  
For **90% confidence**, $ Z_{\alpha/2} = 1.645 $.  

📌 **Step 2:** Compute the **margin of error**  
\begin{align}
\text{Margin of Error} = Z_{\alpha/2} \times \frac{s}{\sqrt{n}}
\end{align}
\begin{align}
= 1.645 \times \frac{2.3}{\sqrt{225}}
\end{align}
\begin{align}
= 1.645 \times 0.153 = 0.252
\end{align}

📌 **Step 3:** Compute the **confidence interval**  
\begin{align}
\text{Lower bound} = 11.6 - 0.252 = 11.35
\end{align}
\begin{align}
\text{Upper bound} = 11.6 + 0.252 = 11.85
\end{align}

🔹 **Final Answer: $ (11.35, 11.85) $**  


✅ **Interpretation:**  
“We are **90% confident** that the **true mean number of unoccupied seats per flight** is **between 11.35 and 11.85**.” ✈️  

---

## **🔹 Key Takeaways**
✅ **Confidence intervals** provide a **range** that likely contains the population mean $ \mu $.  
✅ **Z-Statistic** is used when **$ \sigma $ is known**, otherwise, we use **$ s $**.  
✅ **Larger samples** produce **narrower confidence intervals**.  
✅ **Higher confidence levels** produce **wider intervals**.  
✅ **Margin of error** tells us how much uncertainty is in our estimate.  

---

## **🔹 Python Implementation**
Want to automate this? Here’s how you can calculate a **95% confidence interval** in Python:

In [4]:
import scipy.stats as stats
import numpy as np

# Given data
x_bar = 230  # Sample mean
sigma = 40  # Population standard deviation (if known)
n = 100  # Sample size
confidence = 0.95  # Confidence level

# Find the z-critical value
z_critical = stats.norm.ppf((1 + confidence) / 2)

# Compute the margin of error
margin_of_error = z_critical * (sigma / np.sqrt(n))

# Compute confidence interval
lower_bound = x_bar - margin_of_error
upper_bound = x_bar + margin_of_error

print(f"📊 95% Confidence Interval: ({lower_bound:.2f}, {upper_bound:.2f})")

📊 95% Confidence Interval: (222.16, 237.84)


---

🎯 **Final Thought:**  
Confidence intervals **add meaning** to data by accounting for uncertainty! Now you know how to estimate population means with confidence. 🔥 🚀

# **📌 Confidence Interval for a Population Mean: Student’s t-Statistic**
  
We previously used the **z-distribution** for confidence intervals when the **sample size was large** ( $ n \geq 30 $ ) or the **population standard deviation $ \sigma $ was known**.  

But what happens when:  
✅ **The sample size is small ( $ n < 30 $ )**  
✅ **The population standard deviation $ \sigma $ is unknown**  

In these cases, we **cannot** use the z-distribution. Instead, we use **Student’s t-distribution**, which **adjusts for extra variability** when estimating $ \sigma $ using the sample standard deviation $ s $. 🎯  


---

## **🔹 The Student’s t-Distribution**
The **t-distribution** is:  
- **Mound-shaped and symmetric** (like the normal distribution).  
- **More spread out** than the normal distribution for small samples.  
- **Defined by degrees of freedom (df)**, where $ df = n - 1 $.  
- **As $ n $ increases, the t-distribution approaches the normal (z) distribution**.  

📌 **Formula for a Small-Sample Confidence Interval for $ \mu $:**  
\begin{align}
\bar{x} \pm t_{\alpha/2} \times \frac{s}{\sqrt{n}}
\end{align}
Where:  
- $ \bar{x} $ = Sample mean  
- $ s $ = Sample standard deviation  
- $ n $ = Sample size  
- $ t_{\alpha/2} $ = Critical t-value from the **t-table** for confidence level **$ 1 - \alpha $** and **$ df = n - 1 $**  


---

## **🔹 Example 6.4: Estimating Blood Pressure Increase**
A **pharmaceutical company** is testing a **new drug** and needs to estimate the **average increase in blood pressure** caused by the drug.  

📌 **Given data:**  
- **Sample size**: $ n = 6 $  
- **Sample mean**: $ \bar{x} = 2.283 $  
- **Sample standard deviation**: $ s = 0.783 $  
- **Confidence level**: **95%**  

📌 **Step 1:** Find the **t-critical value**  
- Degrees of freedom: $ df = n - 1 = 6 - 1 = 5 $  
- From the **t-table**, for **95% confidence** and $ df = 5 $:  
  - **$ t_{0.025, 5} = 2.571 $**

📌 **Step 2:** Compute the **margin of error**  
\begin{align}
\text{Margin of Error} = t_{\alpha/2} \times \frac{s}{\sqrt{n}}
\end{align}
\begin{align}
= 2.571 \times \frac{0.783}{\sqrt{6}}
\end{align}
\begin{align}
= 2.571 \times 0.3197
\end{align}
\begin{align}
= 0.822
\end{align}

📌 **Step 3:** Compute the **confidence interval**  
\begin{align}
\text{Lower bound} = 2.283 - 0.822 = 1.461
\end{align}
\begin{align}
\text{Upper bound} = 2.283 + 0.822 = 3.105
\end{align}

🔹 **Final Answer: $ (1.461, 3.105) $**  


✅ **Interpretation:**  
“We are **95% confident** that the **true mean increase in blood pressure** caused by the drug is **between 1.461 and 3.105 points**.” 💊  


---

## **🔹 Why Do We Use t Instead of z?**
Using **z-values** assumes we know $ \sigma $, but since we **estimate** $ \sigma $ using $ s $ (which introduces extra variability), the **t-distribution** compensates by being **wider** than the normal distribution.  

💡 **Key Differences:**
- **t-distribution has fatter tails** (more spread-out for small samples).  
- **z-distribution is more concentrated around the mean**.  
- **For large $ n $, $ t $ and $ z $ become nearly identical**.  

📌 **Rule of Thumb:**  
- Use **t-distribution** when **$ n < 30 $ and $ \sigma $ is unknown**.  
- Use **z-distribution** when **$ n \geq 30 $ or $ \sigma $ is known**.  

---

## **🔹 Example 6.5: Estimating Printhead Lifetime**
A **printer manufacturer** tests **15 printheads** to determine the **average number of characters printed before failure**.  

📌 **Given data:**  
- $ n = 15 $  
- $ \bar{x} = 1.239 $ million characters  
- $ s = 0.137 $ million characters  
- **Confidence level**: **99%**  

📌 **Step 1:** Find the **t-critical value**  
- Degrees of freedom: $ df = 15 - 1 = 14 $  
- From the **t-table**, for **99% confidence** and $ df = 14 $:  
  - **$ t_{0.005, 14} = 2.977 $**  

📌 **Step 2:** Compute the **margin of error**  
\begin{align}
\text{Margin of Error} = t_{\alpha/2} \times \frac{s}{\sqrt{n}}
\end{align}
\begin{align}
= 2.977 \times \frac{0.137}{\sqrt{15}}
\end{align}
\begin{align}
= 2.977 \times 0.0354
\end{align}
\begin{align}
= 0.1054
\end{align}

📌 **Step 3:** Compute the **confidence interval**  
\begin{align}
\text{Lower bound} = 1.239 - 0.1054 = 1.134
\end{align}
\begin{align}
\text{Upper bound} = 1.239 + 0.1054 = 1.344
\end{align}

🔹 **Final Answer: $ (1.134, 1.344) $**  


✅ **Interpretation:**  
“We are **99% confident** that the **true mean number of characters printed before failure** is **between 1.134 and 1.344 million characters**.” 🖨️  

---

## **🔹 Python Implementation**
Want to automate this? Here’s how you can compute **t-based confidence intervals** in Python:

In [5]:
import scipy.stats as stats
import numpy as np

# Given data
x_bar = 2.283  # Sample mean
s = 0.783  # Sample standard deviation
n = 6  # Sample size
confidence = 0.95  # Confidence level

# Degrees of freedom
df = n - 1

# Find the t-critical value
t_critical = stats.t.ppf((1 + confidence) / 2, df)

# Compute the margin of error
margin_of_error = t_critical * (s / np.sqrt(n))

# Compute confidence interval
lower_bound = x_bar - margin_of_error
upper_bound = x_bar + margin_of_error

print(f"📊 95% Confidence Interval: ({lower_bound:.3f}, {upper_bound:.3f})")

📊 95% Confidence Interval: (1.461, 3.105)


---

## **🔹 Key Takeaways**
✅ Use **t-distribution** when **$ n < 30 $ and $ \sigma $ is unknown**.  
✅ **Degrees of freedom $ df = n - 1 $** affect the shape of the **t-distribution**.  
✅ **Smaller samples** → **wider confidence intervals** (more uncertainty).  
✅ **Larger samples** → **t-distribution approaches normal (z-distribution)**.  
✅ **Higher confidence levels** → **wider intervals** (more certainty, less precision).

---

## **🔹 Why This Matters?**
Understanding **when to use t vs. z** is **critical** in statistics. In real-world applications (**clinical trials, small studies, quality control**), we often deal with **small samples** and **unknown population parameters**, making the **t-distribution** essential. 🔥 🚀

# **📌 Large-Sample Confidence Interval for a Population Proportion ( $ p $ )**

When estimating **proportions** (such as the percentage of voters supporting a candidate or the fraction of customers preferring a product), we use **confidence intervals for a binomial proportion**.

📌 **Key Assumptions for Large-Sample Proportion Confidence Intervals:**
- The **sample is random**.
- The **sample size is large enough** so that both:
  - $ np \geq 15 $ (at least 15 successes)
  - $ n(1 - p) \geq 15 $ (at least 15 failures)  
  - *If this condition is not met, we must use an adjusted method.*

---

### **🔹 Formula for a Large-Sample Confidence Interval for $ p $**
\begin{align}
\hat{p} \pm z_{\alpha/2} \times \sqrt{\frac{\hat{p}(1 - \hat{p})}{n}}
\end{align}

Where:
- $ \hat{p} = \frac{x}{n} $ = **sample proportion** (point estimate of $ p $)
- $ n $ = **sample size**
- $ z_{\alpha/2} $ = **z-value** for the confidence level
- $ \sqrt{\frac{\hat{p}(1 - \hat{p})}{n}} $ = **standard error** of $ \hat{p}

✅ **Common Confidence Levels and $ z_{\alpha/2} $ Values:**
- **90% Confidence** → $ z_{\alpha/2} = 1.645 $
- **95% Confidence** → $ z_{\alpha/2} = 1.960 $
- **99% Confidence** → $ z_{\alpha/2} = 2.576 $

---

## **🔹 Example 6.6: Estimating Proportion of Consumers Preferring a Cereal Brand**
A **food company** wants to estimate the **true proportion of consumers** who prefer its breakfast cereal.

📌 **Given data:**
- **Sample size:** $ n = 1000 $
- **Consumers preferring brand:** $ x = 313 $
- **Sample proportion:**  
  \begin{align}
  \hat{p} = \frac{x}{n} = \frac{313}{1000} = 0.313
  \end{align}
- **Confidence level:** **95%** ( $ z_{\alpha/2} = 1.960 $ )

📌 **Step 1:** Compute the **standard error**
\begin{align}
SE = \sqrt{\frac{\hat{p}(1 - \hat{p})}{n}}
\end{align}
\begin{align}
= \sqrt{\frac{(0.313)(1 - 0.313)}{1000}}
\end{align}
\begin{align}
= \sqrt{\frac{(0.313)(0.687)}{1000}}
\end{align}
\begin{align}
= \sqrt{0.000215}\approx 0.0147
\end{align}

📌 **Step 2:** Compute the **margin of error**
\begin{align}
ME = z_{\alpha/2} \times SE
\end{align}
\begin{align}
= 1.960 \times 0.0147
\end{align}
\begin{align}
= 0.0288
\end{align}

📌 **Step 3:** Compute the **confidence interval**
\begin{align}
\text{Lower bound} = \hat{p} - ME = 0.313 - 0.0288 = 0.284
\end{align}
\begin{align}
\text{Upper bound} = \hat{p} + ME = 0.313 + 0.0288 = 0.342
\end{align}

🔹 **Final Answer:** **$ (0.284, 0.342) $** or **$ (28.4\%, 34.2\%) $**


✅ **Interpretation:**  
“We are **95% confident** that the **true proportion of consumers** who prefer the company's cereal is **between 28.4% and 34.2%**.” 🥣✨


---

## **🔹 Example 6.7: Estimating Consumer Optimism About the Economy**
The **Bureau of Economic and Business Research (BEBR)** wants to estimate the **proportion of Florida consumers** who are optimistic about the economy.

📌 **Given data:**
- **Sample size:** $ n = 484 $
- **Consumers optimistic about the economy:** $ x = 157 $
- **Sample proportion:**  
  \begin{align}
  \hat{p} = \frac{157}{484} = 0.3245
  \end{align}
- **Confidence level:** **90%** ( $ z_{\alpha/2} = 1.645 $ )

📌 **Step 1:** Compute the **standard error**
\begin{align}
SE = \sqrt{\frac{\hat{p}(1 - \hat{p})}{n}}
\end{align}
\begin{align}
= \sqrt{\frac{(0.3245)(1 - 0.3245)}{484}}
\end{align}
\begin{align}
= \sqrt{\frac{(0.3245)(0.6755)}{484}}
\end{align}
\begin{align}
= \sqrt{0.0004525} \approx 0.0213
\end{align}

📌 **Step 2:** Compute the **margin of error**
\begin{align}
ME = z_{\alpha/2} \times SE
\end{align}
\begin{align}
= 1.645 \times 0.0213
\end{align}
\begin{align}
= 0.035
\end{align}

📌 **Step 3:** Compute the **confidence interval**
\begin{align}
\text{Lower bound} = 0.3245 - 0.035 = 0.289
\end{align}
\begin{align}
\text{Upper bound} = 0.3245 + 0.035 = 0.359
\end{align}

🔹 **Final Answer:** **$ (0.289, 0.359) $** or **$ (28.9\%, 35.9\%) $**


✅ **Interpretation:**  
“We are **90% confident** that the **true proportion of Florida consumers** who are optimistic about the economy is **between 28.9% and 35.9%**.” 📈🌞  

✅ **Does this indicate that a minority (<50%) is optimistic?**  
Yes! Since **the entire confidence interval is below 50%**, we **can conclude** that **less than half of Florida consumers are optimistic** about the economy.


---

## **🔹 Adjusted Confidence Interval for Small $ p $ Values**
🔸 **Problem:** The **normal approximation fails** when $ p $ is very small (e.g., probability of rare workplace injuries).  

🔸 **Solution:** Use an **adjusted proportion**:

\begin{align}
\tilde{p} = \frac{x + 2}{n + 4}
\end{align}

🔸 **Example 6.8: Injury Rate at a Jewelry Store**
- **Sample size:** $ n = 200 $
- **Injured workers:** $ x = 3 $
- **95% confidence interval required**  

📌 **Step 1:** Compute the **adjusted proportion**  
\begin{align}
\tilde{p} = \frac{3 + 2}{200 + 4} = \frac{5}{204} = 0.0245
\end{align}

📌 **Step 2:** Compute the **adjusted standard error**  
\begin{align}
SE = \sqrt{\frac{\tilde{p}(1 - \tilde{p})}{n + 4}}
\end{align}
\begin{align}
= \sqrt{\frac{(0.0245)(0.9755)}{204}}
\end{align}
\begin{align}
= \sqrt{0.000117} \approx 0.0108
\end{align}

📌 **Step 3:** Compute the **confidence interval**
\begin{align}
ME = 1.960 \times 0.0108 = 0.0212
\end{align}
\begin{align}
\text{Lower bound} = 0.0245 - 0.0212 = 0.004
\end{align}
\begin{align}
\text{Upper bound} = 0.0245 + 0.0212 = 0.046
\end{align}

🔹 **Final Answer:** **$ (0.004, 0.046) $** or **$ (0.4\%, 4.6\%) $**  


✅ **Interpretation:**  
“We are **95% confident** that the **true proportion of injured jewelry store workers** is **between 0.4% and 4.6%**.” 💎🛠️

---

## **🔹 Python Implementation**
Want to automate confidence intervals for proportions? Here's how in Python:

In [6]:
import scipy.stats as stats
import numpy as np

n = 1000  # Sample size
x = 313  # Successes
p_hat = x / n
confidence = 0.95

# Compute standard error
SE = np.sqrt((p_hat * (1 - p_hat)) / n)

# Compute margin of error
z_critical = stats.norm.ppf((1 + confidence) / 2)
ME = z_critical * SE

# Compute confidence interval
lower_bound = p_hat - ME
upper_bound = p_hat + ME

print(f"📊 95% Confidence Interval: ({lower_bound:.3f}, {upper_bound:.3f})")

📊 95% Confidence Interval: (0.284, 0.342)


🔥 **Boom!** 🚀